### PR TITLE
create testing file structure

### PR DESCRIPTION
**Pull request by htwight**

### CODE REVIEW

File: app/src/test/kotlin/io/imhungry/di/AppModuleTest.kt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package io.imhungry.di
+
+class AppModuleTest {
+}
```
Comment (TripleFrequency) [inline app/src/test/kotlin/io/imhungry/di/AppModuleTest.kt]: You don't write tests for dependency injection components.

File: app/src/test/kotlin/io/imhungry/di/RoomModuleTest.kt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package io.imhungry.di
+
+class RoomModuleTest {
+}
```
Comment (TripleFrequency) [inline app/src/test/kotlin/io/imhungry/di/RoomModuleTest.kt]: Same here, no tests for dependency injection needed

**Comment by htwight** [comment]
Alright, can I still have a pull request with the new folders so we're on
the same page for testing, or should I submit another pull request once I
have some other things done.

**Comment by TripleFrequency** [comment]
Empty folders are not included in commits, so you can't add the folders without files. Beyond that, you should include these files only once they contain content.

As far as testing is concerned, I think setting this standard is fine, but tests are automatically placed in the same locations as these files whenever you generate them in the IDE. If people don't do that, it will hopefully get caught in a code review.

I don't believe this needs its own PR at this point.

**Pull request closed without merging**